src/core/store/middleware.ts

```diff
@@ -102,7 +124,7 @@ export namespace Middleware {
       reduxBatch,
       applyMiddleware(...middleware),
       reduxBatch,
-      applyMiddleware(thunkEvaluator),
+      applyMiddleware(thunkEvaluator, Middleware.validator()),
```

Comment:
if we have two `Middleware.validator()`s we can just call it once and save the result for reuse